dirsrvtests/tests/suites/clu/dsconf_logging.py

```diff
@@ -0,0 +1,164 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2025 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import json
+import subprocess
+import logging
+import pytest
+from lib389._constants import DN_DM
+from lib389.topologies import topology_st as topo
+
+pytestmark = pytest.mark.tier1
+
+log = logging.getLogger(__name__)
+
+SETTINGS = [
+    ('logging-enabled', None),
+    ('logging-disabled', None),
+    ('mode', '777'),
+    ('compress-enabled', None),
+    ('compress-disabled', None),
+    ('buffering-enabled', None),
+    ('buffering-disabled', None),
+    ('max-logs', '4'),
+    ('max-logsize', '7'),
+    ('rotation-interval', '2'),
+    ('rotation-interval-unit', 'week'),
+    ('rotation-tod-enabled', None),
+    ('rotation-tod-disabled', None),
+    ('rotation-tod-hour', '12'),
+    ('rotation-tod-minute', '20'),
+    ('deletion-interval', '3'),
+    ('deletion-interval-unit', 'day'),
+    ('max-disk-space', '20'),
+    ('free-disk-space', '2'),
+]
+
+DEFAULT_TIME_FORMAT = "%FT%TZ"
+
+
+def execute_dsconf_command(dsconf_cmd, subcommands):
+    """Execute dsconf command and return output and return code"""
+
+    cmdline = dsconf_cmd + subcommands
+    proc = subprocess.Popen(cmdline, stdout=subprocess.PIPE)
+    out, _ = proc.communicate()
+    return out.decode('utf-8'), proc.returncode
+
+
+def get_dsconf_base_cmd(topo):
+    """Return base dsconf command list"""
+    return ['/usr/sbin/dsconf', topo.standalone.serverid,
+            '-j', '-D', DN_DM, '-w', 'password', 'logging']
+
+
+def test_log_settings(topo):
+    """Test each log settings can be set successfully
+
+    :id: ffc912a6-c188-413d-9c35-7f4b3774d946
```

Comment:
duplicate id

Reply:
Yeah meant to fix that but forgot.  This was a side project of mine that I've _slowly_ been working on for the last few months.